packages/ui/core-components/src/lib/unsorted/ui/Footnote.svelte

```diff
@@ -0,0 +1,18 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	export let id = ''; // Footnote identifier (number or letter)
+</script>
+
+<div class="block">
+	<div class="inline-flex items-baseline text-xs mb-2 mt-0">
+		<!-- Footnote number -->
+		<sup class="text-gray-500">{id}</sup>
```

Comment:
I dont think I would expect this to be superscript. The number is superscript where you are referencing, but the footnote itself would have the number inline at the same text size. cf wikipedia